script.js

```diff
@@ -1,7 +1,533 @@
+};
+
+const myFunction3 = () => {
+  document.getElementById('popup_desktop').classList.remove('view');
+  window.scrollTo(0, 1000);
+  document.body.classList.remove('overflow');
+  document.getElementById('sect').classList.remove('blur');
+};
+
+const myFunction4 = () => {
+  document.getElementById('popup_mobile').classList.add('see');
+  window.scrollTo(0, 0);
+  document.body.classList.add('overflow');
+  document.getElementById('sect').classList.add('blur');
+};
+
+const myFunction5 = () => {
+  document.getElementById('popup_mobile').classList.remove('see');
+  window.scrollTo(0, 1000);
+  document.body.classList.remove('overflow');
+  document.getElementById('sect').classList.remove('blur');
+};
+
+const mobile_container = [
+  {
+    name: 'Professional Art Printing Data',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    tech: ['css', 'html', 'bootstrap', 'Ruby'],
+    button: 'See Project',
+  },
+
+  {
+    name: 'Professional Art Printing Data',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    tech: ['css', 'html', 'bootstrap', 'Ruby'],
+    button: 'See Project',
+  },
+
+  {
+    name: 'Professional Art Printing Data',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    tech: ['css', 'html', 'bootstrap', 'Ruby'],
+    button: 'See Project',
+  },
+
+  {
+    name: 'Professional Art Printing Data',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    tech: ['css', 'html', 'bootstrap', 'Ruby'],
+    button: 'See Project',
+  },
+
+  {
+    name: 'Professional Art Printing Data',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    tech: ['css', 'html', 'bootstrap', 'Ruby'],
+    button: 'See Project',
+  },
+];
+
+const projects_container = [
+  {
+    title: 'Multi-Post Stories',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    tech: ['css', 'html', 'bootstrap', 'Ruby'],
+    'image desktop': 'images/Img.png',
+    'image mobile': 'images/image1.png',
+    button: 'See Project',
+    'link to live version': '#1',
+    'link to source': '#2',
+  },
+
+  {
+    title: 'Professional Art Printing Data',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    tech: ['html', 'bootstrap', 'Ruby'],
+    button: 'See Project',
+    'link to live version': '#3',
+    'link to source': '#4',
+  },
```

Comment:
- [ ] You are required to use the same array of objects for all your versions (desktop and mobile) for the page. Each object should include at least name, description, image, tech, links. Currently, you have two variables for your array of objects and each has different keys. Give this a fix.

Reply:
okay I'll do that